lib/pages/portal/service/website_provider.dart

```diff
@@ -132,15 +132,15 @@ class WebsiteProvider with ChangeNotifier {
   /// number of times the user accessed website with ID `websiteIds[i]`.
   void incrementNumberOfVisits(Website website) async {
     website.numberOfVisits++;
-    print(website.numberOfVisits);
+    print('visits:' + website.numberOfVisits.toString());
```

Comment:
Remove this

Reply:
print('visits:' + website.numberOfVisits.toString()); // eu am doar linia asta in cod

Reply:
Yes. Remove it. No point in leaving prints in the code.